delivery_roulier/__manifest__.py

```diff
@@ -20,6 +20,7 @@
     "external_dependencies": {
         "python": [
             "roulier",  # '>0.2.0'
+            "odoo_test_helper",
```

Comment:
I wonder if it is necessary ? Should it not be in a test-requirements file or something ?

Reply:
@yankinmax @florian-dacosta It's subjective. Either, you add it manually to test-requirements.txt, either you put here (and it will be added automatically to requirements.txt).

There is no guideline on that point particularly

Reply:
I would leave it here only for one important reason.
When we aggregate PR's as pending merges on projects we are merging them one by one.
And it can happen there is a `test-requirements.txt` file on a branch.
Then we have a conflict and automated process fails.
You need to complete the process manually and sometimes it leads to mistakes.
So, if its possible to have it directly on `requirements.txt` it would be great
@rousseldenis @florian-dacosta

Reply:
I though there were some guidlines about this, because one could use the module in production without having the test libs installed (which would lead to some warning beeing in the manifest file no?)
Anyway, I don't really care on my side, so ok for me!

Reply:
I don't understand what you mean @simahawk 
It's not a part of migration.
If we should move this dependency to `test-requirements.txt` it's feasible quickly.

Reply:
It's not a dependency. It's a test dependency. Just remove it from here and add it manually to test-req. That's it :wink: